system/jlib/jlzw.cpp

```diff
@@ -2057,6 +2058,28 @@ class CCompressedFileBase : implements ICompressedFileIO, public CInterface
         //Allow the disk read and write to send multiple blocks in a single operation - to reduce cloud io costs.
         assertex(trailer.blockSize);
         numBlocksToBuffer = 1;
+        if (fileio && _bufferSize == (size32_t)-1)
```

Comment:
quibble/not new: "_bufferSize" made me think there was a member called "bufferSize", but there is no clash, I'd rename the arg. to "bufferSize"